doc/02_Pattern sicurezza/05_accesso-del-fruitore/04_id_auth_rest_02.rst

```diff
@@ -125,7 +124,7 @@ Regole di processamento
 Note:
 
 -  In merito agli algoritmi da utilizzare si fa riferimento al capitolo
```

Comment:
```suggestion
```

Reply:
@mfortini fai tu fix con: In merito agli algoritmi da utilizzare si fa riferimento alle Linee Guida sulla sicurezza, emanate dall'Agenzia per l'Italia Digitale ai sensi dell'articolo 71 del decreto legislativo 7 marzo 2005, n. 82 (Codice dell'Amministrazione Digitale).